dds/DCPS/Discovery.cpp

```diff
@@ -13,6 +13,7 @@
 #include "DdsDcpsCoreC.h"
 #include "Marked_Default_Qos.h"
 #include "dds/DCPS/SafetyProfileStreams.h"
+#include "DCPS_Utils.h"
```

Comment:
This is the only line changed in this file.  Is it needed?

Reply:
Yes.  `retcode_to_string` moved to DCPS_Utils.h.